doc/nrf/ug_bootloader_adding.rst

```diff
@@ -175,10 +183,12 @@ To use a custom signing command with this bootloader, set the following options
          -DCONFIG_SB_SIGNING_PUBLIC_KEY=\"/path/to/pub.pem\" \
          -DCONFIG_SB_SIGNING_COMMAND=\"my_command\"
 
-      (Escaped quotations avoid malformed-string warnings from Kconfig)
+      Escaped quotations avoid malformed-string warnings from Kconfig.
+
+.. note::
 
-The public key string must be an absolute path to the location of the public key file.
-Environment variables and the ``~`` character on Unix systems will not be expanded when setting this value from a ``prj.conf`` file or Kconfig fragment, but will be expanded on the command line.
+   The public key string must be an absolute path to the location of the public key file.
```

Comment:
Why don't we reduce copy-paste and reference the next line as an extension to this one:

```suggestion
   The public key string must be an absolute path to the location of the public key file, as discussed in :ref:`ug_bootloader_adding_immutable_keys` above.
```
The next line would then be removed.

Reply:
I agree. Reference earlier material (with a link or reference to specific heading/step) instead of repeating.

Reply:
fixed